change/@fluentui-web-components-4d726fab-94eb-4143-bd3a-ea3ef5886ec2.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix: error in card styles",
```

Comment:
Can you be more specific here and denote that it's a syntax error? Otherwise if we step back through commits we likely won't know without inspecting *what* it was.